### PR TITLE
Leadership

### LINKED ISSUE #46: Implement leadership page structure
For reference, please use: https://www.moleculardevices.com/leadership

leadership-page

- [ ] Create all leader profiles
- [ ] Implement lightbox functionality for leader details

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #46

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/leadership
- After: https://leadership--moleculardevices--hlxsites.hlx.page/leadership

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -1,31 +1,355 @@
...
-    object-fit: cover;
-}
\ No newline at end of file
+  aspect-ratio: 4 / 3;
+  object-fit: cover;
+  width: 100%;
+}
+
+/* MODAL */
+.modal,
```
Comment (mhaack) [inline blocks/cards/cards.css:34]: Are these generic modal styles used on other places as well or specific to the leadership cards? If generic lets move to styles.css. If specific lets scope with some extra class.

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -1,31 +1,355 @@
...
+  padding-top: 5px;
+}
+
+.modal-body .cards-card-body p:first-child {
+  margin-top: 0;
+}
+
+.modal-carousel-item {
+  display: none;
+  /* -webkit-transition: .6s ease-in-out left;
```
Comment (mhaack) [inline blocks/cards/cards.css:144]: Remove commented css

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -1,31 +1,355 @@
...
+    transform: translateX(100%);
+  }
+
+  100% {
+    transform: translateX(0);
+  }
+}
+
+
+/* ================ Leadership Page ================ */
```
Comment (mhaack) [inline blocks/cards/cards.css:226]: This should be "leadership block option" as it is specific to the block not the page.

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -1,31 +1,355 @@
...
+h1#leadership {
```
Comment (mhaack) [inline blocks/cards/cards.css:227]: Is this is only for this page? Of do we have other `h1` using similar styles. A block should not define style outside of the block itself.

File: blocks/cards/cards.js [hunk 5]
```diff
@@ -15,4 +15,190 @@ export default function decorate(block) {
...
+    removeActiveClassFromArr(modalCarouselItems, 'active');
+
+    if (activeID === itemsLength) {
+      document.getElementById(0).classList.add('active');
+    } else {
+      document.getElementById(activeID + 1).classList.add('active');
+    }
+  }
+
+  // document
```
Comment (mhaack) [inline blocks/cards/cards.js:190]: Remove commented code

**Comment by dev-rajneeshkumar** [comment]
Hi @mhaack , we have made all the fixes and PR is ready for final review. Kindly check!

**Review by mhaack — APPROVED** [review]

**Pull request merged**